data/styles/classic.mplstyle

```diff
@@ -246,15 +246,6 @@ path.simplify_threshold: 0.1111111111111111
 path.snap: True
 path.sketch: None
 
-savefig.dpi: 100
-savefig.facecolor: w
-savefig.edgecolor: w
-savefig.format: png
-savefig.bbox: standard
```

Comment:
Never really noticed these, but yeah those shouldn't be in the stylesheet. Savefig options should be of course defined in the export figure dialog